config.toml.dist

```diff
@@ -51,3 +51,10 @@ secret =
 discovery_url = "https://accounts.google.com/.well-known/openid-configuration"
 # Google OpenID Connect id_token issuer domain - Default: "accounts.google.com"
 issuer_domain = "accounts.google.com"
+
+
+[limits]
+# Limit the number of session of any kind.
+auth = ["per-second:3:1:1", "per-minute:20:60:5"]
+# Limit the number of email loop authentications.
+auth_email = ["per-second:1:1:1", "per-minute:10:60:5"]
```

Comment:
Just as a confirmation, when reading through that patch I though the same, that it would be nice to make that easier. I already have issues understanding the bucket system (never encountered that before), adding granularity on top of that in the config does not help.